qlib/model/interpret/base.py

```diff
@@ -0,0 +1,33 @@
+#  Copyright (c) Microsoft Corporation.
+#  Licensed under the MIT License.
+
+"""
+Interfaces to interpret models
+"""
+
+import pandas as pd
+from abc import abstractmethod
+
+
+class FeatureInt:
+    """Feature (Int)erpreter"""
+
+    @abstractmethod
+    def get_feature_importance(self) -> pd.Series:
+        ...
```

Comment:
Give more information about the return value in the docstring. 
- The index is the feature name.
- The greater the value, the higher importance.